cypress/e2e/views/migration-wave.view.ts

```diff
@@ -12,6 +12,7 @@ export enum MigrationWaveView {
     waveStatusColumn = "td[data-label='Status']",
     yearInput = "input[aria-label='Select year']",
     migrationWavesTable = "table[aria-label='Migration waves table']",
+    manageApplicationsTable = "table[aria-label='Migration waves table']",
```

Comment:
There are already 2 selectors in the migration waves view pointing to the same place `migrationWavesTable` and `waveTable` please reuse one of these.